heron/metricsmgr/src/java/com/twitter/heron/metricsmgr/sink/FileSink.java

```diff
@@ -64,25 +64,20 @@
 
   @Override
   public void init(Map<String, Object> conf, SinkContext context) {
-    filenamePrefix = (String) conf.get(FILENAME_KEY) + "." + context.getMetricsMgrId();
+    filenamePrefix = conf.get(FILENAME_KEY) + "." + context.getMetricsMgrId();
```

Comment:
What's the contract for conf? I saw most of places don't check the existence. E.g.,  https://github.com/twitter/heron/blob/master/heron/common/src/java/com/twitter/heron/common/config/SystemConfig.java#L468